### PR TITLE
fix:polish

### LINKED ISSUE #191: Add message to create a deck when saving a word if a user has no decks
Capture d’écran 2020-04-24 à 09 49 16

### PR DESCRIPTION
**Pull request by blendaf**
fixes #191

### CODE REVIEW

File: components/browse-decks/BrowseDecks.js [hunk 1]
```diff
@@ -28,7 +27,7 @@ const DeckWrapper = styled.div`
   @media screen and (max-width: ${screenSizes.smallPhone.max}) {
     justify-content: center;
     align-items: center;
-    margin-bottom: 20px;
+    padding-bottom: 20px;
```
Comment (juliuscc) [inline components/browse-decks/BrowseDecks.js]: Is this necessary?

File: components/exchange/useSearchTranslation.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ const fetchTranslation = (word, from, to) => {
 
   return fetch(url, {
     method: 'post',
+    // mode: 'no-cors',
```
Comment (juliuscc) [inline components/exchange/useSearchTranslation.js]: I think this is a mistake

File: components/browse-decks/BrowseDecks.js (hunk 1 above)
Comment (blendaf) [reply components/browse-decks/BrowseDecks.js]: Nope but I just thought that it didn't matter to change it back

File: components/exchange/useSearchTranslation.js (hunk 2 above)
Comment (blendaf) [reply components/exchange/useSearchTranslation.js]: Okay but I think that is something you did

Comment (juliuscc) [reply components/exchange/useSearchTranslation.js]: Yes I remember doing it. Could you remove it from the PR?

**Review by juliuscc — APPROVED** [review]
Perfect!

**Pull request merged**